### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Vaibhav-afk**
### Updated 
- **nord.svg** 
- **contributing.md** 
- **readme.md**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Our aim is to **keep it simple** for the developers to contribute to this projec
 5. [Generate your personal access token](https://github.com/settings/tokens)
 
 6. Create a `.env` file in root directory and in there write  
-   `TOKEN=<your_generated_token>`
```
Comment (Ashutosh00710) [inline CONTRIBUTING.md:32]: Don't change this line @Vaibhav-afk , this (<your_generated_token>) is a standard representation of replacement

**Review by Ashutosh00710 — CHANGES_REQUESTED** [review]
All other changes are good, one suggestion is there, please read the [contributing.md](https://github.com/Ashutosh00710/github-readme-activity-graph/blob/main/CONTRIBUTING.md). 

If you are adding a theme, place a screenshot of that theme with your pull request message too.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Vaibhav-afk) [reply CONTRIBUTING.md:32]: > Don't change this line @Vaibhav-afk , this (<your_generated_token>) is a standard representation for replacement

Okay.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Our aim is to **keep it simple** for the developers to contribute to this projec
 5. [Generate your personal access token](https://github.com/settings/tokens)
 
 6. Create a `.env` file in root directory and in there write  
-   `TOKEN=your_generated_token`
+   `TOKEN=<your_generated_token>`
```
Comment (Ashutosh00710) [inline CONTRIBUTING.md]: Everything looks goods now.

**Review by Ashutosh00710 — APPROVED** [review]
Merging it to main.

**Comment by Ashutosh00710** [comment]
@Vaibhav-afk  Lucent Theme is now available in production 🎉

**Pull request merged**